pkg/controller/kogitoruntime/kogitoruntime_controller.go

```diff
@@ -112,6 +112,10 @@ func (r *ReconcileKogitoRuntime) Reconcile(request reconcile.Request) (result re
 		return
 	}
 
+	if err = infrastructure.MountProtoBufConfigMapOnDataIndex(r.client, instance); err != nil {
+		return
```

Comment:
shouldn't it print some warning message or debug if config map fails to mount?

Reply:
I had added Error msg here